docs/www/Jenkinsfile

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env groovy
+/*
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+// email configuration
+def mailSender = 'robot@numberfour.eu'
+def String recipient = 'brian.smith@numberfour.eu'
+def String replyTo = 'brian.smith@numberfour.eu'
+def String team = 'brian.smith@numberfour.eu, staff-devtools@numberfour.eu'
+def String buildhtml = env.BUILD_URL+"execution/node/5/ws/sourceX/docs/www/web-html"
```

Comment:
The mail config should be de-personalized. Cf. mail config in https://github.com/eclipse/n4js/blob/master/Jenkinsfile